hedera-node/hedera-smart-contract-service-impl/src/test/java/com/hedera/node/app/service/contract/impl/test/exec/gas/TinybarValuesTest.java

```diff
@@ -27,46 +26,73 @@
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
-import org.mockito.Mock;
 import org.mockito.junit.jupiter.MockitoExtension;
 
 @ExtendWith(MockitoExtension.class)
 class TinybarValuesTest {
     private static final ExchangeRate RATE_TO_USE =
             ExchangeRate.newBuilder().hbarEquiv(2).centEquiv(14).build();
     private static final long RBH_FEE_SCHEDULE_RICE = 77_000L;
```

Comment:
what is a `_RICE`?  Some amount of tinybar clumped together into a not-so-tiny _grain_?

Reply:
Fixed.